src/controllers/accommodation.controller.js

```diff
@@ -0,0 +1,45 @@
+import AccommodationService from '../services/accommodation.service';
+import ResponseService from '../services/response.service';
+
+/**
+ *
+ *
+ * @class AccommodationController
+ */
+class AccommodationController {
+  /**
+           *
+           *
+           * @static
+           * @param {req} req
+           * @param {res} res
+           * @returns {response} @memberof AccommodationController
+           */
+  static async createAccommodation(req, res) {
+    const { dataValues } = await AccommodationService.createAccommodation({ ...req.body });
+    const { updatedAt, createdAt, ...newAccommodation } = dataValues;
+    ResponseService.setSuccess(201, 'Accommodation is successfully created', newAccommodation);
+    return ResponseService.send(res);
+  }
+
+  /**
+           *
+           *
+           * @static
+           * @param {req} req
+           * @param {res} res
+           * @returns {response} @memberof AccommodationController
+           */
+  static async getAccommodationType(req, res) {
+    const accommodationtypes = await AccommodationService.findAllType();
+    const availableAccommodationTypes = accommodationtypes.map((type) => {
+      const { dataValues } = type;
+      const { updatedAt, createdAt, ...accommodationType } = dataValues;
```

Comment:
If you want to omit the updatedAt and createdAt properties, you should do that at the database level by defining a custom select property in sequelize find method.